packages/circuit-ui/components/Hr/Hr.tsx

```diff
@@ -21,12 +21,13 @@ const baseStyles = ({ theme }: StyleProps) => css`
   label: hr;
   display: block;
   width: 100%;
-  border: 1px solid ${theme.colors.n300};
+  border: 0;
```

Comment:
Do we need `border: 0;` at all here?

Reply:
Yes, because the `<hr />` element has a default border.

Reply:
Ah, makes sense